TPL/force-app/main/default/flows/Parent_Flow_For_Account.flow-meta.xml

```diff
@@ -219,7 +219,7 @@
         <recordTriggerType>CreateAndUpdate</recordTriggerType>
         <triggerType>RecordAfterSave</triggerType>
     </start>
-    <status>Active</status>
```

Comment:
Authorized deactivation of replaced flows; will approve destruction changes after testing processes and final deployments are completed.